journal/202110.md

```diff
@@ -0,0 +1,450 @@
+
+<a id="dcrwallet" />
+
+**[dcrwallet](https://github.com/decred/dcrwallet)**
+
+_dcrwallet is a wallet server used by command-line and graphical wallet applications._
+
+- implemented [`getblockheader`](https://github.com/decred/dcrwallet/pull/2098) and [`getcurrentnet`](https://github.com/decred/dcrwallet/pull/2102) methods in SPV mode (to be used by DCRDEX)
+- added [`spv`](https://github.com/decred/dcrwallet/pull/2094) field to `walletinfo` result to distinguish between syncing modes
+- fixed [sync loss](https://github.com/decred/dcrwallet/pull/2099) in SPV mode
+
+<a id="decrediton" />
+
+**[Decrediton](https://github.com/decred/decrediton)**
+
+_Decrediton is a full-featured desktop wallet app with integrated voting, StakeShuffle mixing, Lightning Network, DEX trading, and more. It runs with or without a full blockchain (SPV mode)._
+
+User-facing changes:
+
+- implemented new UI design on [LN Overview](https://github.com/decred/decrediton/pull/3551) tab. Wallet, Network, and Watchtowers tabs have been grouped under a newly introduced Advanced tab.
+- new UI design for wallet [modals](https://github.com/decred/decrediton/pull/3534) with added password visibility toggle
+- added the ability to use [random VSP](https://github.com/decred/decrediton/pull/3560) in Automatic Ticket Purchases (with a configurable maximum fee)
+- automaticalle enable the new [per-account](https://github.com/decred/decrediton/pull/3579) encryption for new wallets
+- improved usability of the [VSP selector](https://github.com/decred/decrediton/pull/3563)
+- ~13 bug fixes
+
+Internal:
+
+- updated DEX integration via [libdexc](https://github.com/decred/decrediton/pull/3549) to utilize upcoming 0.3 features (localized UI, restore from seed, and discovery of existing accounts)
+- updated to [Electron 15](https://github.com/decred/decrediton/pull/3571)
+- added a new GUI tool for [translating](https://github.com/decred/decrediton/pull/3569) strings and a [guide](https://github.com/decred/decrediton/blob/master/app/i18n/community_translators.md) for community translators
+- automated tests for [Accounts](https://github.com/decred/decrediton/pull/3577) views
+
+{ image candidates: 3534 3551 3560 }
+
+<a id="politeia" />
+
+**[Politeia](https://github.com/decred/politeia)**
+
+_Politeia is Decred's proposal system. It is used to request funding from the Decred treasury._
+
+User-facing changes:
+
+- added standard way to show any status change reason (not only censored but now also abandoned and closed)
+- support [multiple](https://github.com/decred/politeiagui/pull/2629) billing status changes. Ideally there should be only one (from Active to Completed or Closed), but this allows to correct admin mistakes. It is also possible to temporarily [disable](https://github.com/decred/politeia/pull/1533) status changes.
+- improved [identity error](https://github.com/decred/politeiagui/pull/2623) messages
```

Comment:
This improves and standardizes the identity error.  The same error is now displayed anytime the user attempts to write data to politeia without their active identity being loaded in the browser.  The error message directs them to their user details page to fix the issue and provides a link to do so.

Reply:
Good detail. Added a slightly compressed version.

A side effect of such mentions is it helps to raise [awareness](https://github.com/decred/dcrdesign/issues/261) about user identity.